GetandTake/Helpers/ImageLinkHtmlHelperExtension.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Web.Mvc;
+
+namespace GetandTake.Helpers;
+
+public static class ImageLinkHtmlHelperExtension
+{
+    public static MvcHtmlString ProduceImageLink(int categoryImageId, string linkText)
```

Comment:
Try to check this comment: https://stackoverflow.com/a/39537346
Looks like ​`MvcHtmlString​` is deprecated